scrapegraphai/graphs/screenshot_scraper_graph.py

```diff
@@ -0,0 +1,71 @@
+"""
+ScreenshotScraperGraph Module
+"""
+
+from typing import Optional
+import logging
+from pydantic import BaseModel
+from .base_graph import BaseGraph
+from .abstract_graph import AbstractGraph
+
+from ..nodes import (
+    FetchScreenNode,
+    GenerateAnswerFromImageNode,
+)
+
+class ScreenshotScraperGraph(AbstractGraph):
+    """
+   smart_scraper.run()
+        )
+    """
+
+    def __init__(self, prompt: str, source: str, config: dict, schema: Optional[BaseModel] = None):
+        super().__init__(prompt, config, source, schema)
+
+
+    def _create_graph(self) -> BaseGraph:
+        """
+        Creates the graph of nodes representing the workflow for web scraping.
+
+        Returns:
+            BaseGraph: A graph instance representing the web scraping workflow.
+        """
+        fetch_screen_node = FetchScreenNode(
+            input="url",
+            output=["imgs"],
+            node_config={
+                "link": self.source
+            }
+        )
+        generate_answer_from_image_node = GenerateAnswerFromImageNode(
+            input="doc",
+            output=["parsed_doc"],
```

Comment:
same goes here, neither the input nor output arguments are really used by the node